cli/src/test/functions-test.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
```

Comment:
In `devonfw-ide` this file was included by all individual tests and therefore named this way.
If this is not the case here, couldn't we simply move this code to `all-tests.sh` or rename it (e.g. to `all-tests-functions.sh`)?

Reply:
Renamed to `all-tests-functions.sh` now.